src/AppBundle/Controller/CommitteeEventController.php

```diff
@@ -34,21 +37,52 @@ public function showAction(CommitteeEvent $event): Response
      * @Method("GET|POST")
      * @Entity("event", expr="repository.findOneBySlug(slug)")
      */
-    public function attendAction(CommitteeEvent $event): Response
+    public function attendAction(Request $request, CommitteeEvent $event): Response
     {
+        $committee = $event->getCommittee();
+
+        $command = new CommitteeEventRegistrationCommand($event, $this->getUser());
+        $form = $this->createForm(CommitteeEventRegistrationType::class, $command);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $this->get('app.committee.committee_event_registration_handler')->handle($command);
+            $this->addFlash('notice', $this->get('translator')->trans('committee.event.registration.success'));
+
+            return $this->redirectToRoute('app_committee_attend_event_confirmation', [
+                'committee_uuid' => (string) $committee->getUuid(),
+                'slug' => $event->getSlug(),
+                'uuid' => (string) $command->getRegistrationUuid(),
+            ]);
```

Comment:
Why not using something like `$this->redirect($this->get('app.twig.committee_extension')->getUrl('app_committee_attend_event_confirmation', $committee))`?